pkg/store/redis/store.go

```diff
@@ -86,9 +140,17 @@ func (rs *redisStore) UpdateStatus(scanJobID string, newStatus job.ScanJobStatus
 		scanJob.Error = error[0]
 	}
 
+	if rs.isTerminalStatus(newStatus) {
+		return rs.saveScanJobWithExpire(*scanJob, rs.cfg.ScanJobTTL)
+	}
+
 	return rs.SaveScanJob(*scanJob)
 }
 
+func (rs *redisStore) isTerminalStatus(status job.ScanJobStatus) bool {
+	return status == job.Finished || status == job.Failed
+}
```

Comment:
Similarly here, I'm not sure it's worth an extra function - it's a little bit harder to read IMO

Reply:
Provided that we do not add any new terminal statuses (99,9% likely) I agree!